.travis.yml

```diff
@@ -73,6 +73,8 @@ jobs:
         - travis_wait 80 make emulator-ndebug -C regression SUITE=JtagDtmSuite JVM_MEMORY=3G
         - travis_wait 80 make emulator-jtag-dtm-tests-32 -C regression SUITE=JtagDtmSuite JVM_MEMORY=3G
         - travis_wait 80 make emulator-jtag-dtm-tests-64 -C regression SUITE=JtagDtmSuite JVM_MEMORY=3G
+        - travis_wait 80 make emulator-jtag-dtm-tests-32 -C regression SUITE=JtagDtmSuite JTAG_DTM_TEST="MemTest64 MemTest32 MemTest8" JTAG_DTM_ENABLE_SBA=on JVM_MEMORY=3G
```

Comment:
Nice. Let's keep an eye on the Travis runtime for this and make sure it's not excessive, otherwise more tests is better!